app/src/main/java/app/revanced/manager/ui/viewmodel/InstallerViewModel.kt

```diff
@@ -56,14 +56,20 @@ import java.util.logging.LogRecord
 class InstallerViewModel(
     private val input: Destination.Installer
 ) : ViewModel(), KoinComponent {
+    private val logger = ManagerLogger()
     private val app: Application by inject()
     private val fs: Filesystem by inject()
     private val pm: PM by inject()
     private val workerRepository: WorkerRepository by inject()
     private val installedAppRepository: InstalledAppRepository by inject()
     private val rootInstaller: RootInstaller by inject()
 
-    val packageName: String = input.selectedApp.packageName
+    internal var packageInstallerResult by mutableStateOf<PackageInstallerResult?>(null)
+        private set
+
+    internal var showInstallerStatusDialog by mutableStateOf(false)
+
+    internal val packageName: String = input.selectedApp.packageName
```

Comment:
```suggestion
    var packageInstallerResult by mutableStateOf<PackageInstallerResult?>(null)
        private set

    var showInstallerStatusDialog by mutableStateOf(false)

    val packageName: String = input.selectedApp.packageName
```
Is internal necessary?